server/dataset/tests.py

```diff
@@ -19,7 +19,7 @@ def setUp(self):
         user.save()
         self.client = APIClient()
         response = self.client.post(
-            "/auth-token/", {"username": username, "password": password}, format="json",
+            "/api/v1/auth-token/", {"username": username, "password": password}, format="json",
```

Comment:
I believe it would be best to store the `/api/v1` path on a variable, so that it's easier and less error-prone to edit in the future. Also, it would probably be better to have the current full version be stored on a single place, and use that to build the api path.

This comment applies to all the edits in this file.